### PR TITLE
Validate oc keys

### PR DESCRIPTION
**Pull request by cschweipert**

### CODE REVIEW

File: mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb [hunk 1]
```diff
@@ -5,3 +5,11 @@ def oc_code
 event :clear_jurisdiction_key_cache do
   Card::Region.cache.reset_all
 end
+
+event :validate_jurisdiction_key, :validate,
+      on: :save, changed: :content do
+  oc_key = Card.search left: { type: :region },
+                       right: :oc_jurisdiction_key,
+                       content: card.content
+  errors.add :content, "OC key already exists" if oc_key.present?
```
Comment (ethn) [inline mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb]: Let's write out "Open Corporates".

File: mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb [hunk 2]
```diff
@@ -5,3 +5,11 @@ def oc_code
 event :clear_jurisdiction_key_cache do
   Card::Region.cache.reset_all
 end
+
+event :validate_jurisdiction_key, :validate,
+      on: :save, changed: :content do
+  oc_key = Card.search left: { type: :region },
+                       right: :oc_jurisdiction_key,
+                       content: card.content
```
Comment (ethn) [inline mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb:13]: Technically this search will return an array (a list of cards). So even though it doesn't matter, it's slightly clearer to say `oc_keys`.

**Review by ethn — APPROVED** [review]
This looks very good. Thank you!

File: mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb (hunk 2 above)
Comment (cschweipert) [reply mod/wikirate_companies/set/type_plus_right/region/oc_jurisdiction_key.rb:13]: Technically there should be only one element in the array. If I change it, oc_keys.present seems slightly less clear.

**Pull request merged**